.openvex/templates/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenVEX Templates Directory
```

Comment:
Can you point to how common adding this detail is across other projects? Want to make sure our project is consistent in usage across the org as well as that the maintainers who will continue to maintain support for this after-the-fact understand what this is used for,

Reply:
I believe having these details across other projects will be good.

Reply:
@jonathan-innis: This PR was made as part of the Cloud Native Security Slam and the files in this PR was generated using https://github.com/openvex/vexctl, the instructions to which are given in this [doc](https://docs.google.com/document/d/1TLahqW6OpWQXbaVJ-SR8z-o0yLSoGKYTHxPU0PyyjXo/edit#heading=h.kjt6lttp3u0h). To answer your question about consistency, I believe it'd be the same across all projects in the org since the `.openvex` directory is generated with a script.